Demos/Basic-ObjC/Basic-ObjC/Assets/forms.json

```diff
@@ -1,372 +1,404 @@
-                  "type":"field",
-                  "action":"update"
-                }
-              ],
-              "validations":{
-                "max_value": 100,
-                "min_value": 10
-              }
-            },
-            {
-              "id":"total",
-              "title":"Total",
-              "type":"float",
-              "size":{
-                "width":25,
-                "height":1
-              },
-              "disabled":true,
-              "formula":"$base_salary + $bonus"
-            }
-          ]
-        },
-        {
-          "id":"companies",
-          "type":"dynamic",
-          "action_title":"ADD COMPANY ✛",
-          "fields":null
-        },
-        {
-          "id":"contacts",
-          "type":"dynamic",
-          "action_title":"ADD CONTACT ✛",
-          "fields":null
-        }
-      ]
-    }
-  ],
-  "templates":{
-    "fields": null,
-    "sections":[
-      {
-        "id":"companies",
-        "fields":[
-          {
-            "id":"companies[:index].name",
-            "title":"Company Name",
-            "type":"name",
-            "validations":{
-              "required":true,
-              "min_length":2
-            },
-            "size":{
-              "width":50,
-              "height":1
-            }
-          },
-          {
-            "id":"companies[:index].phone_number",
-            "title":"Phone number",
-            "type":"number",
-            "size":{
-              "width":30,
-              "height":1
-            }
-          },
-          {
-            "id":"companies[:index].remove",
-            "title":"Remove",
-            "type":"button",
-            "size":{
-              "width":20,
-              "height":1
-            }
-          }
-        ]
-      },
-      {
-        "id":"contacts",
-        "fields":[
-          {
-            "id":"contacts[:index].name",
-            "title":"Contact name",
-            "type":"name",
-            "validations":{
-              "required":true,
-              "min_length":2
-            },
-            "size":{
-              "width":50,
-              "height":1
-            }
-          },
-          {
-            "id":"contacts[:index].phone_number",
-            "title":"Phone number",
-            "type":"number",
-            "size":{
-              "width":30,
-              "height":1
-            }
-          },
-          {
-            "id":"contacts[:index].remove",
-            "title":"Remove",
-            "type":"button",
-            "size":{
-              "width":20,
-              "height":1
-            }
-          }
-        ]
-      }
-    ]
-  }
-}
+	"groups": [
+			   {
+			   "id": "personal-details",
+			   "title": "Personal Details",
+			   "sections": [
+							{
+							"id": "personal-details-0",
+							"fields": [
+									   {
+									   "id": "first_name",
+									   "title": "First name",
+									   "info": "Fornavn",
```

Comment:
Sure, I ran it in JSONlint.com which is probably why the whole formatting has changed :/